src/tests/ll_verification.py

```diff
@@ -723,6 +723,269 @@ def ll_ddi_adv_bv_21_c(transport, upperTester, lowerTester, trace):
+        PrimMinInterval = toArray(advInterval, 3)
+        PrimMaxInterval = toArray(advInterval, 3)
+        PrimChannelMap  = 0x07  # Advertise on all three channels (#37, #38 and #39)
+        OwnAddrType     = SimpleAddressType.PUBLIC
+        PeerAddrType    = SimpleAddressType.PUBLIC
+        PeerAddress     = toArray(0x456789ABCDEF, 6)
+        FilterPolicy    = AdvertisingFilterPolicy.FILTER_NONE
+        TxPower         = 0
+        PrimAdvPhy      = PhysicalChannel.LE_1M
+        SecAdvMaxSkip   = 0
+        SecAdvPhy       = phy
+        Sid             = 0
+        ScanReqNotifyEnable = 0
+
+        success = success and preamble_ext_advertising_parameters_set(transport, upperTester, Handle, Properties, PrimMinInterval, PrimMaxInterval,
+                                                      PrimChannelMap, OwnAddrType, PeerAddrType, PeerAddress, FilterPolicy, TxPower,
+                                                      PrimAdvPhy, SecAdvMaxSkip, SecAdvPhy, Sid, ScanReqNotifyEnable, trace)
+        if not success:
+            return success
+
+        if round.DataLength > 0:
+            # Set adv data
+            maxFragmentSize = 251
+            advData = random.randint(1, 255, round.DataLength)
+            remainingAdvData = advData[:]
+            firstFragment = True
+            while (len(remainingAdvData) > 0):
+                if firstFragment:
+                    if len(remainingAdvData) <= maxFragmentSize:
+                        op = FragmentOperation.COMPLETE_FRAGMENT
+                    else:
+                        op = FragmentOperation.FIRST_FRAGMENT
+                else:
+                    if len(remainingAdvData) <= maxFragmentSize:
+                        op = FragmentOperation.LAST_FRAGMENT
+                    else:
+                        op = FragmentOperation.INTERMEDIATE_FRAGMENT
+                endIndex = maxFragmentSize if len(remainingAdvData) >= maxFragmentSize else len(remainingAdvData)
+                status = le_set_extended_advertising_data(transport, upperTester, Handle, op, round.FragmentPref, remainingAdvData[:endIndex], 100)
+                if status != 0:
+                    return False
+                remainingAdvData = remainingAdvData[endIndex:]
+                firstFragment = False
+        else:
+            advData = []
+            status = le_set_extended_advertising_data(transport, upperTester, Handle, FragmentOperation.COMPLETE_FRAGMENT, round.FragmentPref, advData, 100)
+            if status != 0:
+                return False
+
+        flush_events(transport, upperTester, 100)
+        success = success and preamble_ext_advertise_enable(transport, upperTester, Advertise.ENABLE, [Handle], [round.Duration], [round.MaxEvents], trace)
+
+        if not success:
+            return success
+
+        if round.Duration != 0:
+            # Wait until end of duration + 500ms (to make sure advertising has stopped)
+            transport.wait(5500)
```

Comment:
I think the magic `5500` should be ``
```suggestion
            transport.wait(round.Duration*10 + 500)
```

Reply:
Done